visallo/visallo
# Editing visibility (4.0)

Testing Instructions:
- Make sure users that don't have PUBLISH privilege can't edit the element visibility
- Make sure users with PUBLISH privilege can edit the element visibility
- Make sure "element" metadata (`Modified By`, `Modified Date`) is in the "i" visibility popover

Points of Regression: 

CHANGELOG
Fixed: Users without publish privileges being able to edit the element visibility
Fixed: Element metadata not displaying in "i" popover
